test-data/unit/cmdline.test

```diff
@@ -130,30 +130,6 @@ two/mod/__init__.py: note: See https://mypy.readthedocs.io/en/stable/running_myp
 two/mod/__init__.py: note: Common resolutions include: a) using `--exclude` to avoid checking one of them, b) adding `__init__.py` somewhere, c) using `--explicit-package-bases` or adjusting MYPYPATH
 == Return code: 2
 
-[case testFlagsFile]
-# cmd: mypy @flagsfile
```

Comment:
There doesn't seem to be any other test case that uses `@flagsfile`, so this test case should not be deleted. We should just change it to use some other flags.